package.json

```diff
@@ -4,12 +4,12 @@
   "version": "0.0.0",
   "scripts": {
     "build": "lerna run build --stream",
-    "build:demo": "build-storybook -s packages/bedrock/dist -o demo",
+    "build:demo": "storybook build -o demo",
     "format": "prettier-package-json --write && lerna run format --stream",
     "postinstall": "husky install",
     "lint": "eslint scripts/*.js && lerna run lint --stream",
     "prepare": "yarn build",
-    "start": "start-storybook -s packages/bedrock/dist -p 6006",
+    "start": "storybook dev -p 6006",
```

Comment:
Why were the static directories removed?

Reply:
They weren't, the `-s` (`--static-dirs`) has been deprecated and the configuration has been moved, [see here](https://storybook.js.org/docs/react/api/main-config-static-dirs#page-top).